channeldb/channel.go

```diff
@@ -2068,11 +2072,25 @@ type HTLC struct {
 	// from the HtlcIndex as this will be incremented for each new log
 	// update added.
 	LogIndex uint64
+
+	// ExtraData contains any additional information that was transmitted
+	// with the HTLC via TLVs. This data *must* already be encoded as a
+	// TLV stream, and may be empty.
+	ExtraData []byte
```

Comment:
Nit: the comment of this struct says: `// HTLC is the on-disk representation of a hash time-locked contract` but with this new change it seems to me that this has changed, basically the on-disk representation compared to the in-memory representation diverge a bit because we now have this ExtraData which is still part of the `Onion-Blob` on disk. Maybe add a comment here that its not quite the same?